CMakeLists.txt

```diff
@@ -85,6 +85,13 @@ set(NETWORK "netlrts" CACHE STRING "Target network layer, can be one of mpi,mult
 set(TARGET "charm++" CACHE STRING "Target build type, can be one of charm++,AMPI,LIBS,charm4py,ChaNGa,everylb,all-test")
 set(ARCH "" CACHE STRING "Target architecture, can be one of i386,x86_64,arm7,arm8,ppc64le") # By default, detect arch we are running on.
 
+if(NOT ${ARCH} STREQUAL ${CMAKE_HOST_SYSTEM_PROCESSOR})
```

Comment:
Does the statement work without the `${}`?

Reply:
Fixed.